### PR TITLE
Talisker status endpoints

### PR DESCRIPTION
**Pull request by earnubs**
See http://talisker.readthedocs.io/en/latest/contract.html#endpoints

### CODE REVIEW

File: src/server/routes/status.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    memcached.end();
+  });
+};
+
+export const error = () => {
+  throw new Error('This is a servicestatus error test');
+};
+
+export const metric = (req, res) => {
+  res.send(200);
```
Comment (bartaz) [inline src/server/routes/status.js]: It's just a placeholder for now? How does it differ from `ping`?

Comment (earnubs) [inline src/server/routes/status.js]: The route has middleware on it which will send a metric to statsd, the handler is just sending a default OK; would a comment here help?

File: test/routes/t_login.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import proxyquire from 'proxyquire';
 import loginRoutes from '../../src/server/routes/login.js';
 
 const routesStub = {
+  status: () => {},
```
Comment (bartaz) [inline test/routes/t_login.js]: Not sure if it is relevant here (maybe we need whole server), but in other tests we found a way to test a single route without creating an instance of whole server and empty routes stub: https://github.com/canonical-ols/javan-rhino/blob/master/test/routes/t_api.js#L14-L18

May be worth updating now (or at some point).

Comment (earnubs) [inline test/routes/t_login.js]: I tried this very quickly and it didn't work, so I'll put it in another PR later.
